eq-author-api/schema/tests/comments.test.js

```diff
@@ -91,6 +93,120 @@ describe("comments", () => {
     });
   });
 
+  it("should edit a comment on question page and then query that comment", async () => {
+    await createComment(ctx, {
+      pageId: createdQuestionPage.id,
+      commentText: "a new comment is created",
+    });
+
+    const queryNewComment = await queryComments(ctx, {
+      pageId: createdQuestionPage.id,
+    });
+    const commentId = queryNewComment.comments[0].id;
+
+    await updateComment(ctx, {
+      pageId: createdQuestionPage.id,
+      commentId: commentId,
+      commentText: "an edited comment",
+    });
+
+    const queryEditedComment = await queryComments(ctx, {
+      pageId: createdQuestionPage.id,
+    });
```

Comment:
Same here I believe. As the `updateComment` and `createComment` is returning the updated comment data.